docs/nidigital/rep_caps.rst

```diff
@@ -12,16 +12,25 @@
 Repeated Capabilities
 =====================
 
-    Repeated capabilities attributes are used to set the `channel_string` parameter to the
-    underlying driver function call. This can be the actual function based on the :py:class:`Session`
-    method being called, or it can be the appropriate Get/Set Attribute function, such as :c:`niDigital_SetAttributeViInt32()`.
+    :py:class:`nidigital.Session` supports "Repeated Capabilities", which are multiple instances of the same type of
+    functionality. The repeated capabilities supported by :py:class:`nidigital.Session` are:
 
-    Repeated capabilities attributes use the indexing operator :python:`[]` to indicate the repeated capabilities.
-    The parameter can be a string, list, tuple, or slice (range). Each element of those can be a string or
-    an integer. If it is a string, you can indicate a range using the same format as the driver: :python:`'0-2'` or
-    :python:`'0:2'`
+    #. channels_
+    #. pins_
+    #. instruments_
+    #. pattern_opcode_events_
+    #. conditional_jump_triggers_
+    #. sites_
+    #. rio_events_
```

Comment:
I'm not sure about this... are the RIO triggers and events usable from Python at all in an application?

Reply:
They are. We test it: https://github.com/ni/nimi-python/blob/2986696c11dd4b64b50b3e41744a7dc55ecc1f6b/src/nidigital/system_tests/test_system_nidigital.py#L146

Reply:
They work. But are they actually useful in an application? Given that there is no LVFPGA support in Python.

Reply:
Oh, maybe not.
I can create an issue to remove them.

Reply:
>They work. But are they actually useful in an application? Given that there is no LVFPGA support in Python.

I think there is support.
https://github.com/ni/nifpga-python